src/SixLabors.Fonts/TextOptions.cs

```diff
@@ -9,9 +9,9 @@
 namespace SixLabors.Fonts
 {
     /// <summary>
-    /// Provides configuration options for rendering text.
+    /// Provides configuration options for rendering and shaping text.
     /// </summary>
-    public sealed class TextOptions
+    public class TextOptions
```

Comment:
I've kept `TextOptions` as the name here. I want to use `TextDrawingOptions` in ImageSharp.Drawing